src/styles/panel.scss

```diff
@@ -42,7 +42,7 @@
   height: 100%;
   border-left: 1px solid var(--color-base-300);
   border-top-right-radius: var(--border-radius-sm);
-  z-index: 100;
+  z-index: 11;
```

Comment:
I would go lower but the journal entries table header has z-idx = 10. At some point we should reevaluate every z index in our components